Header header


float32 range_min
float32 range_max